Directory.Packages.props

```diff
@@ -12,7 +12,7 @@
     <PackageVersion Include="Google.Protobuf" Version="[3.19.4,4.0)" />
     <PackageVersion Include="Grpc" Version="[2.44.0,3.0)" />
     <PackageVersion Include="Grpc.Net.Client" Version="[2.52.0,3.0)" />
-    <PackageVersion Include="Grpc.Tools" Version="[2.44.0,3.0)" />
+    <PackageVersion Include="Grpc.Tools" Version="[2.56.2,3.0)" />
```

Comment:
Just making sure we have clarity, is this considered "prod package" and why? Why would we want to update the version?

Reply:
OTLP Exporter consumes `Grpc.Tools` package but it's marked with `PrivateAssets="all"` so it doesn't show up as a package dependency.

> Why would we want to update the version?

`Grpc.Tools` package doesn't have a dependency on any other packages. It wouldn't hurt to update the version and get its latest changes considering its usage is more of development harness.

Reply:
Got it, sounds like this package needs to be moved to nonprod?

Reply:
Doesn't block this PR though.